jest-preset.json

```diff
@@ -8,7 +8,6 @@
   "transform": {
     "^.+\\.(ts|js|html)$": "<rootDir>/node_modules/jest-preset-angular/preprocessor.js"
   },
-  "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|js)$",
   "moduleFileExtensions": [
```

Comment:
Ugh, but Jest will not match `*.ts` files by default. Why not changing it to `testMatch`?

Reply:
hmm, but if we add `testMatch`, people who want to use `testRegex` won't be able to run the tests. I think we should put `testMatch` to the README inside installation setup.

Reply:
But `testRegex` is already set and we write about it in README, so it will be the same situation. I'm not going to remove the default. One can always override it without any issues. Both cannot work with each other and Jest will warn you about it:

Reply:
ah ok then I'll adjust `testRegex` to be `testMatch` 👍